### PR TITLE
Fix #40 standardised all pages to p-6 except pages with scanners. also optimised search function for smaller screens

### LINKED ISSUE #40: Standardise margin or padding

### PR DESCRIPTION
**Pull request by shaunBoahCodes**

### CODE REVIEW

File: src/AuthTemplate.vue [hunk 1]
```diff
@@ -6,7 +6,7 @@ import LoadingView from '@/components/Utilities/LoadingView.vue'
 <template>
   <div class="flex flex-col h-screen w-full">
     <NavBar />
-    <div class="mx-3 grow flex">
+    <div class="grow p-6 flex">
```
Comment (cweitat) [inline src/AuthTemplate.vue:9]: please use margin by default

> In general, use margins when you're adjusting the spacing of an element in relation to another element (i.e a div in relation to another div on the page), and padding when you're adjusting the look of an individual element (i.e the amount of pixels between the edge of a div and the text within it).

File: src/App.vue [hunk 2]
```diff
@@ -6,8 +6,8 @@ import LoadingView from '@/components/Utilities/LoadingView.vue'
 
 <template>
   <div class="flex flex-col h-screen">
-    <div class="mx-3 grow flex">
-      <RouterView class="grow flex flex-col" />
+    <div class="grow flex">
+      <RouterView class="grow p-6 flex flex-col" />
```
Comment (cweitat) [inline src/App.vue:10]: same

**Review by cweitat — CHANGES_REQUESTED** [review]

**Review by cweitat — APPROVED** [review]

File: src/App.vue [hunk 3]
```diff
@@ -6,8 +6,8 @@ import LoadingView from '@/components/Utilities/LoadingView.vue'
 
 <template>
   <div class="flex flex-col h-screen">
-    <div class="mx-3 grow flex">
-      <RouterView class="grow flex flex-col" />
+    <div class="grow flex">
```
Comment (cweitat) [inline src/App.vue:9]: purpose of grow flex and grow flex again in routerview?

File: src/AuthTemplate.vue [hunk 4]
```diff
@@ -6,7 +6,7 @@ import LoadingView from '@/components/Utilities/LoadingView.vue'
 <template>
   <div class="flex flex-col h-screen w-full">
     <NavBar />
-    <div class="mx-3 grow flex">
+    <div class="grow m-6 flex">
```
Comment (cweitat) [inline src/AuthTemplate.vue:9]: same here, double grow flex in here and router view

Comment (shaunBoahCodes) [reply src/AuthTemplate.vue:9]: this one is necessary to keep the router view centered

**Review by cweitat — APPROVED** [review]

**Pull request merged**